uiowa/uiowa
# Started adding uiowa_auth module to uiowa

Closes #2630 
# How to test


1. composer install
2. drush @sandbox.local cr
2. blt ds --site=sandbox.uiowa.edu
3. drush @sandbox.local uli
4. make sure uiowa_auth is enabled
